hummingbird/ml/supported.py

```diff
@@ -167,6 +174,8 @@ def _build_sklearn_operator_list():
             VarianceThreshold,
         ]
 
+        return [x for x in supported_ops if x is not None]
```

Comment:
Was there some bug?

Reply:
Imputer is None if sklearn >= 0.22. I can add a comment.